cli/src/vote.rs

```diff
@@ -78,11 +78,16 @@ pub fn process_create_vote_account(
     )?;
     let required_balance =
         rpc_client.get_minimum_balance_for_rent_exemption(VoteState::size_of())?;
+    let lamports = if required_balance > 0 {
```

Comment:
if all wallets have to do this, maybe the bank should instead

Reply:
It's ultimately a system CreateAccount instruction. Are you suggesting that any CreateAccount attempt where lamports=0 be upped to 1 on processing?

Reply:
no, I'm saying that if we have a "get_minimum_balance_for_rent_exemption()" RPC, maybe it should go ahead and do the work of 1.max(rent_calculator.get_exemption(data.len())

CC @ParthDesai

Reply:
Oh, gotcha! Yes, that would work nicely.

Reply:
@rob-solana, I think this is not something get_minimum_balance should do. 

Reason being, that we are only suggesting minimum lamports required to make account rent exempt (according to rent_calculator), not amount of lamports you need to create account with.

Reply:
fair enough